src/Apps/Juno/Command.hs

```diff
@@ -19,17 +19,22 @@ import Data.Aeson (encode, object, (.=))
 import Data.Aeson.Encode.Pretty (encodePretty)
 
 import Juno.Types (CommandEntry(..), CommandResult(..))
+import Juno.Types.Base (RequestId)
```

Comment:
Not really a fan of qualified imports from project-local files unless necessary.